views/src/index.ejs

```diff
@@ -21,5 +21,21 @@
         </ul>
       </div>
+      <div class="jumbotron info">
+        <p class="h1">Etkinliklerimiz</p>
+        <hr class="my-4">
+        <p class="h2">Programlama Yarışması</p>
+        <p class="lead">ODTÜ Bilgisayar Topluluğu, üniversite öğrencilerini bilgisayar bilimlerine teşvik etmek amacıyla, her sene Türkiye genelinde bir programlama yarışması düzenlemektedir. Geçtiğimiz yıl 20.si gerçekleşen yarışmamızın 21.si iki aşamalı olarak 23-26 Mart ve 13 Nisan 2018 tarihlerinde gerçekleşmesi planlanlanmaktadır. Yarışmaya katılmak için en fazla üçer kişiden oluşan öğrenci grupları, ön eleme sorularını çözerek yarışmamızın resmi başvuru ve kayıt sayfası olan <a href="http://yarisma.cclub.metu.edu.tr">http://yarisma.cclub.metu.edu.tr</a> adresine cevaplarını gönderirler.</p>
+        <img class="leftimage rounded" src="/img/yarisma.jpg" alt="Programlama Yarışması-Bilişim Günü">
+        <img class="rightimage rounded" src="/img/bilisim.jpg" alt="Programlama Yarışması-Bilişim Günü">
+        <p class="h2">Bilişim Günü</p>
+        <p class="lead">21. Geleneksel Programlama Yarışması’nın ödül töreni ise 14 Nisan 2018 tarihinde gerçekleşmesi planlanan “Bilişim Günü”nde yapılmaktadır. ODTÜ Kültür Kongre Merkezi'nde gerçekleşecek bu etkinlik kapsamında, yazılım sektörünün büyük şirketlerinin katıldığı seminerler düzenlenecektir. Eğlenceli ve bilgilendirme amaçlı olan seminer ve sunumların yanı sıra ödüllü çekilişler de yapılmaktadır. Günün sonunda ise ödül töreni gerçekleştirilip ilk 3’e giren gruplara para ödülü verilecektir.</p>
+        <p class="h2">Teoriden Pratiğe</p>
+        <p class="lead">Öğrencilere eğitim hayatları boyunca öğretilen teorik bilgilerin onlarda uyandırdığı “Ben bu bilgileri iş hayatında nasıl kullanacağım?” ve benzeri soruları cevaplamayı amaçlayan, sektörden seçkin insanların tecrübelerini ve bu teorik bilgileri nasıl kullandıklarını anlattıkları konuşmalardan oluşması planlanan “Teoriden Pratiğe” adlı etkinliğimizi ilk kez 2 Aralık 2017 tarihinde düzenleyeceğiz. Etkinlikte Arçelik, ING Bank, Taleworlds Entertainment ve hala iletişimde olduğumuz şirketlerden katılımcılar yer alacak. Her yıl düzenli olarak düzenlemeyi planladığımız bu etkinliğin, hedef kitlemiz olan 3-4. sınıfta okuyan mühendislik ve beşeri bilimler öğrencileri için faydalı ve yol gösterici olacağına inanıyoruz.</p>
+        <p class="h2">S-Talk</p>
+        <p class="lead">Dönemde 3-4 kez düzenlemeyi planladığımız bu etkinlikler, bilgisayar ve teknoloji alanında bilgili/tecrübeli kişiler tarafından verilecek eğitici kısa konuşmalar fikriyle çıktı. İlkini 26 Ekim 2017’de “Reinforcement Learning” alanında gerçekleştirdik. ODTÜ Bilgisayar Mühendisliği bölümünden iki asistanımız ICTAI 2017 konferansından davet alan makalelerini bizlere sundu. Sıradaki S-talk’lar için şirketler ve eğitimcilerle görüşmelerimiz devam ediyor.</p>
```

Comment:
s/davet/kabul/

Reply:
content is just placeholder, just to be more relevant than lorem ipsum